telegraf/telegraf.conf

```diff
@@ -14,7 +14,7 @@
   omit_hostname = false
 
 [[outputs.influxdb]]
-  urls = ["http://influxdb:8086"]
+  urls = ["http://influxdb:8086/api/v2"]
```

Comment:
This seems to be incompatible with InfluxDB v1.8.  It breaks all of the existing controls trying to access the InfluxDB data sources.  This should be combined with a complete project update to InfluxDB v2.x.